ipv-api/src/test/java/uk/gov/di/authentication/ipv/lambda/IPVAuthorisationHandlerTest.java

```diff
@@ -277,11 +278,11 @@ private void usingValidClientSession() {
                 .thenReturn(withAuthenticationRequest().toParameters());
         when(clientSession.getVtrList())
                 .thenReturn(
-                        List.of(
-                                VectorOfTrust.of(
-                                        CredentialTrustLevel.LOW_LEVEL, LevelOfConfidence.NONE),
-                                VectorOfTrust.of(
-                                        CredentialTrustLevel.LOW_LEVEL,
+                        VtrList.of(
+                                new VectorOfTrust(
+                                        CredentialTrustLevel.MEDIUM_LEVEL, LevelOfConfidence.NONE),
+                                new VectorOfTrust(
+                                        CredentialTrustLevel.MEDIUM_LEVEL,
```

Comment:
How come these have changed?

Reply:
A low CTL and and medium LoC is not valid. This worked previously because the vtr was just a list object with all of it's effective methods on the VectorOfTrust class.

Reply:
True, though there's not need to do an identity check if the VTR contains Cl. I can change it back anyway.